packages/vscode-extension/test/suite/envChecker/adapters/testAdapter.ts

```diff
@@ -26,6 +31,21 @@ export class CustomDotnetInstallScript {
   }
 }
 
+// This class is for mocking dotnet-install script path.
+export class CustomPathDotnetInstallScript implements ICustomDotnetInstallScript {
+  private readonly scriptPath: string;
+  /**
+   * @param targetPath: the dir that contains the dotnet-install scripts.
```

Comment:
targetPath?

Reply:
😅  forget to change this when refactoring